index.html

```diff
@@ -5,8 +5,53 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Microverse Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
 </head>
 <body>
+    <header>
+        <nav id="mobile-navbar">
+            <img src="assets/menu.png" alt="">
+        </nav>
+    </header>
+    <section id="headline-section">
+        <div class="container">
+            <h1 >
+                <span id="name-heading">
+                    Hey there<br>
+                I'm Abdul Salam
+                </span>
+                <span id="job-role">
+                    I',m a Software<br>
+                    Developer
+                </span>
+                
+            </h1>        
+            <p>
+                I can help you build a product, feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hesitate to contact me.
+              </p>
+           
```

Comment:
> > - [ ] I would like to acknowledge and appreciate the effort you put into implementing the changes suggested by the previous reviewer. It shows your dedication and commitment to improving the website.👍
> 
> > However, upon reviewing the latest version, I noticed that there is an issue with the spacing at the top of the headline section.😌 The current spacing does not align closely with the Figma design, which is an important guideline for visual consistency throughout the website. 
> 
>> Matching the design as closely as possible is crucial as it helps create a cohesive and professional user experience. Consistency across different pages and components of the website contributes to seamless navigation and enhances overall usability.
> 
>> I kindly request you address this spacing issue and ensure that it aligns more closely with the Figma design.😊
> 
> > > Figma design👇:
>  
> ![sp1](https://github.com/AbdulSalamAbro/portfolio-website/assets/40009354/7b1d7250-119b-440f-8e67-4990b645651d)
>  
> Your design👇:
>  
> ![sp](https://github.com/AbdulSalamAbro/portfolio-website/assets/40009354/094dcf4e-85cd-4d15-9203-62f1290d15f1)
>

Reply:
Hi @Johnadibe the space is working fine in my browser kindly use min 375px size as which is recommended in project requirements